### ISSUE TITLE
Updated OneNote: runtime error occurred during the loading of the COM Add-in.

### ISSUE THREAD

**Issue by itjhowe** [stevencohn/OneMore#439]
This began occurring after I updated OneNote to the latest version. The Addin no longer loads at all. I reinstalled, rebooted, tried the 32 and 64 bit versions of the OneMore installer.

**To Reproduce**
1. Go To Options in OneNote,
2. Go to "Add-ins"
3. Set the "Manage:" field to "COM Add-ins"
4. Click "Go..."
5. Highlighting the "OneMoreAddin" shows the title error next to "Load Behavior"

**Expected behavior**
The Add-In Loads

**Screenshots**
![image](https://user-images.githubusercontent.com/98054695/153534197-511de863-0b2f-437f-8a69-06a534bbad7b.png)

**Environment (if applicable):**
 - OneNote Version: 365 for enterprise
 - OneMore Version Whatever the latest version is

**Additional context**
Add any other context about the problem here. Attach page XML file or the app log file as appropriate. You can find the log file here: %localappdata%\temp\OneMore.log

**Comment by itjhowe** [comment]
[OneMoreSetup.log](https://github.com/stevencohn/OneMore/files/8051622/OneMoreSetup.log)

Installed with Admin account

**Comment by stevencohn** [comment]
Note to self
* OneMoreSetup.log shows normal install (and expected TrustedProtocol access exception during uninstall)
* OneMore..log included

```
63528:1] Running command AnalyzeCommand
63528:1] Error running command AnalyzeCommand
63528:1] System.OutOfMemoryException
Message: Out of memory.
Source: System.Drawing
StackTrace:    at System.Drawing.Image.GetThumbnailImage(Int32 thumbWidth, Int32 thumbHeight, GetThumbnailImageAbort callback, IntPtr callbackData)
   at River.OneMoreAddIn.Commands.AnalyzeCommand.ReportImage(Table detail, XElement image, Boolean printout)
   at River.OneMoreAddIn.Commands.AnalyzeCommand.ReportPage(Table table, String pageId)
   at River.OneMoreAddIn.Commands.AnalyzeCommand.ReportPages(XElement container, XElement section, String folderPath, String skipId)
   at River.OneMoreAddIn.Commands.AnalyzeCommand.ReportAllPages(XElement container, XElement folder, String folderPath, String skipId)
   at River.OneMoreAddIn.Commands.AnalyzeCommand.<Execute>d__20.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at River.OneMoreAddIn.CommandFactory.<Run>d__6.MoveNext()
TargetSite: [System.Drawing] System.Drawing.Image::GetThumbnailImage()
```

and 

```
60780:6] ..error invoking action
60780:6] ..System.Runtime.InteropServices.COMException
Message: Exception from HRESULT: 0x8004202C
Description: The selection is invalid
ErrorCode: 0x8004202C (-2147213268)
HResult: 0x8004202C (-2147213268)
Source: Microsoft.Office.Interop.OneNote
StackTrace:    at Microsoft.Office.Interop.OneNote.ApplicationClassCOM.UpdatePageContent(String bstrPageChangesXmlIn, DateTime dateExpectedLastModified, XMLSchema xsSchema, Boolean force)
   at Microsoft.Office.Interop.OneNote.ApplicationClass.UpdatePageContent_AutoBusyRetry(String bstrPageChangesXmlIn, DateTime dateExpectedLastModified, XMLSchema xsSchema, Boolean force)
   at Microsoft.Office.Interop.OneNote.ApplicationClass.UpdatePageContent(String bstrPageChangesXmlIn, DateTime dateExpectedLastModified, XMLSchema xsSchema, Boolean force)
   at River.OneMoreAddIn.OneNote.<>c__DisplayClass50_0.<Update>b__0()
   at River.OneMoreAddIn.OneNote.<InvokeWithRetry>d__26.MoveNext()
TargetSite: [Microsoft.Office.Interop.OneNote] Microsoft.Office.Interop.OneNote.ApplicationClassCOM::UpdatePageContent()

60780:6] Running command InsertExpandCommand
60780:15] Running command RemoveCitationsCommand
60780:14] Running command SearchAndReplaceCommand
60780:14] ..found 43 matches
60780:5] Running command CropImageCommand
60780:4] Running command ColorizeCommand
60780:4] Running command ColorizeCommand
60780:4] Running command SettingsCommand
60780:4] Error running command SettingsCommand
60780:4] System.ComponentModel.Win32Exception
Message: The handle is invalid
Source: System.Windows.Forms
StackTrace:    at System.Windows.Forms.Control.GetSafeHandle(IWin32Window window)
   at System.Windows.Forms.Form.ShowDialog(IWin32Window owner)
   at River.OneMoreAddIn.Commands.SettingsCommand.<Execute>d__1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at River.OneMoreAddIn.CommandFactory.<Run>d__6.MoveNext()
TargetSite: [System.Windows.Forms] System.Windows.Forms.Control::GetSafeHandle()
```

**Comment by stevencohn** [comment]
Note the OutOfMemoryException related to GetThumbnailImage is solved by #607

**Comment by stevencohn** [comment]
Ref #622
